streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -300,17 +299,16 @@ public void shouldThrowExceptionSettingStateListenerNotInCreateState() {
 
     @Test
     public void testNumberDefaultMetrics() {
-        final KafkaStreams streams = createKafkaStreams();
+        props.put(StreamsConfig.NUM_STREAM_THREADS_CONFIG, "1");
```

Comment:
Are these changes related to this PR? They don't seem to be

Reply:
No. That was the fix for the failing test we extracted into an own PR (this fix is also in another of my PRs -- will rebase and clean it up)